pkg/uefi/flashregionsection.go

```diff
@@ -11,33 +11,23 @@ import (
 	"strings"
 )
 
-// FlashRegionSectionSize is the size of the Region descriptor. It is made up by 18 fields, each 16-bits large.
-const FlashRegionSectionSize = 36
+// FlashRegionSectionSize is the size of the Region descriptor. It is made up by 16 fields, each 2x16-bits large.
+const FlashRegionSectionSize = 64
```

Comment:
Does this depend on what version of PCH you're running. What happens when the BIOS targets a newer or older PCH?